extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/util/AuthZUtils.scala

```diff
@@ -68,6 +68,20 @@ private[authz] object AuthZUtils {
     method.invoke(obj, values: _*)
   }
 
+  def invokeAs[T](
+      obj: AnyRef,
+      methodName: String,
+      args: (Class[_], AnyRef)*): T = {
+    Try {
+      invoke(obj, methodName, args: _*)
+    } match {
+      case Success(value) => value.asInstanceOf[T]
```

Comment:
should be better to add these in invoke

Reply:
ok,  exception throwing moved to the `invoke` method.